pwa/src/services/auth.tsx

```diff
@@ -0,0 +1,36 @@
+import { navigate } from "gatsby-link";
+import jwtDecode, { JwtPayload } from "jwt-decode";
+
+export const isBrowser = () => typeof window !== "undefined";
+
+export const getUser = () =>
+  isBrowser() && window.sessionStorage.getItem("user") ? JSON.parse(window.sessionStorage.getItem("user")) : {};
+
+export const setUser = (user) => window.sessionStorage.setItem("user", JSON.stringify(user));
+
+export const handleLogin = (data) => {
+  return setUser(data);
+};
+
+export const isLoggedIn = () => {
```

Comment:
None of the functions have a return value (which is set after the **(): typeHere**).

In this case we're returning a `boolean`, so this should be rewritten to:

```
export const isLoggedIn = (): boolean => ...
```